cluster-autoscaler/cloudprovider/cloud_provider.go

```diff
@@ -57,6 +57,9 @@ type CloudProvider interface {
 	// GetResourceLimiter returns struct containing limits (max, min) for resources (cores, memory etc.).
 	GetResourceLimiter() (*ResourceLimiter, error)
 
+	// Close cleans up open resources before the cloud provider is destroyed, i.e. go routines etc.
+	Close() error
```

Comment:
nit: In case there's no difference, could we rename this `Close` to `Cleanup` so that there are less func names to remember? 😉

Reply:
ok, will update this with the renamed interface method soon.

Reply:
@mmerrill3 Thanks for catching it!

I'd appreciate it if you could also rebase this onto the latest master so that hopefully @MaciekPytel can merge this earlier before this gets conflicted again 😃

Reply:
I am handling rebasing now.